mathesar_ui/src/systems/table-view/table-inspector/record/RowActions.svelte

```diff
@@ -22,22 +23,20 @@
   export let selection: TabularDataSelection;
   export let columnsDataStore: ColumnsDataStore;
 
-  let isDeleting = false;
+  const isDeleting = false;
```

Comment:
I would suggest deleting is variable and its usages altogether.

Reply:
sure, will do it.

Reply:
Fixed it. Please take a look.